pkg/node/manager/manager_test.go

```diff
@@ -74,23 +75,23 @@ func AddrOrPrefixToIP(ip string) (netip.Prefix, error) {
 	return prefix, err
 }
 
-func (i *ipcacheMock) Upsert(ip string, hostIP net.IP, hostKey uint8, k8sMeta *ipcache.K8sMetadata, newIdentity ipcache.Identity) (bool, error) {
+func (i *ipcacheMock) Upsert(ip string, hostIP net.IP, hostKey uint8, k8sMeta *ipcache.K8sMetadata, newIdentity ipcache.Identity, aux ...ipcache.IPMetadata) (bool, error) {
```

Comment:
Dumb question but `aux` naming is for auxiliary?

Reply:
That's correct, it stands for "auxiliary" metadata :+1: